src/liballoc/lib.rs

```diff
@@ -70,7 +70,6 @@
        test(no_crate_inject, attr(allow(unused_variables), deny(warnings))))]
 #![no_std]
 #![needs_allocator]
-#![cfg_attr(not(stage0), deny(warnings))]
```

Comment:
Oh, whoops, yes I was just doing a grep removal, but that doesn't make sense here. Will fix.